index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <style>
+      @media screen and (max-width: 700px){
+        p{
+          font-size: 20px;
+        }
+        body
+        {
+            display: flex;
+        }
+      }
+    </style>
```

Comment:
I recommend that you keep your style definition in a separate CSS file: Styling multiple elements can affect your page's size and download time.

Reply:
Good job on making this recommendation. 💪